packages/plugins/plugin-meeting/src/peers.stories.tsx

```diff
@@ -149,6 +148,97 @@ export default meta;
 type Story = StoryObj<typeof Render>;
 
 export const Default: Story = {
+  render: Render,
+  args: {
+    videoSrc: video,
+  },
+};
+
+export const RePull: Story = {
+  render: (args) => {
```

Comment:
better to pull this out into a local function i think; makes the meta more readable.